### PR TITLE
feat: sample section

### PR DESCRIPTION
**Pull request by padawannn**

### CODE REVIEW

**Comment by Jesus89** [comment]
I made a change to use `tail` instead of `head` because it contains more data.

File: do-catalog/src/views/DatasetData.vue [hunk 1]
```diff
@@ -225,22 +186,27 @@ export default {
...
 
-      if (this.tooltip.left < -20) {
-        this.tooltip.left = -20;
-      } else if (tableBoundingSize.width - this.tooltip.left < -20) {
-        this.tooltip.left = tableBoundingSize.width + 20;
-      }
+      // if (this.tooltip.left < -20) {
+      //   this.tooltip.left = -20;
+      // } else if (tableBoundingSize.width - this.tooltip.left < -20) {
+      //   this.tooltip.left = tableBoundingSize.width + 20;
```
Comment (neokore) [inline do-catalog/src/views/DatasetData.vue:208]: Good simplification! I have to test it in my browsers but it looks good!
BTW, remove all unneeded code to show its real beauty :wink:

File: do-catalog/src/store/modules/data/catalog.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const catalog = {
     datasetsListCount: 0,
     dataset: {},
     keyVariables: {},
-    variables: {},
+    variables: [],
```
Comment (neokore) [inline do-catalog/src/store/modules/data/catalog.js:13]: Why this change?
Is it tested?

**Comment by Jesus89** [comment]
Thanks for pointing this @neokore.

Yep, I tested and it works as expected. Let's cover these details in the next PR.

File: do-catalog/src/views/DatasetData.vue [hunk 3]
```diff
@@ -1,12 +1,17 @@
 <template>
-  <div class="grid grid-cell u-flex__justify--center">
+  <div v-if="!isFetching" class="grid grid-cell u-flex__justify--center">
```
[the remark this answers is not in the document]
Comment (Jesus89) [reply do-catalog/src/views/DatasetData.vue:2]: I think the loader has never be implemented. Let's focus on the new features, and cover this in the future (next iteration)

File: do-catalog/src/views/DatasetData.vue (hunk 1 above)
Comment (Jesus89) [reply do-catalog/src/views/DatasetData.vue:208]: True that. :fire:

File: do-catalog/src/styles/components/_scrollbar.scss [hunk 4]
```diff
@@ -4,18 +4,21 @@ $scrollbar-track-color: #EEE;
 // Chrome / Safari / Opera / Edge
 ::-webkit-scrollbar {
   width: 4px;
+  height: 8px;
 }
 
 ::-webkit-scrollbar-track {
   background-color: $scrollbar-track-color;
+  border-radius: 4px;
```
[the remark this answers is not in the document]
Comment (Jesus89) [reply do-catalog/src/styles/components/_scrollbar.scss:12]: Good catch, but let's worry about this in the next iteration.

File: do-catalog/src/store/modules/data/catalog.js (hunk 2 above)
Comment (Jesus89) [reply do-catalog/src/store/modules/data/catalog.js:13]: Yep, actually, the backend returns an array with the variables.
